01who_we_are/02values.md

```diff
@@ -7,31 +7,73 @@ We take our core values very seriously. They're the underpinning for what we do,
+- Fun stickers! Our sparkly Drupal ponies continue to be a favorite even today.
+- We respond to customer support timely and cheerfully, even the grumpy ones. We're human too, and we get that people have bad days. We like to try to make them a little better when we can.
+
 
 **Work together**
 
-We keep open minds and listen to each other to find common ground and consensus. We support and help each other in ways that make our entire team stronger.
+We believe in Open Source, not only in open software but also in the principles of collaboration and sharing. We fully engage with Open Source communities through contribution and have incorporated these fundamentals throughout our company. We keep open minds and listen to each other to find common ground and improve each other’s work without judgment. We apply these same principles when working with our customers to solve their problems.
+
+Some examples of this value in action are:
+
+- Sharing our Employee Handbook publicly
+- Collaborating on the Drupal 8 User Guide and sharing our enhancements (adding videos)
+
 
 **Build trust**
 
-We build trust through openness, respect, and forgiveness. We treat each other like adults and genuinely care for people. We value our distributed work environment, and we trust our colleagues to make the best use of that flexibility to cultivate the best life balance for them.
+We build trust through openness, respect, and forgiveness. We treat people like equals and genuinely care for them, whether they are part of our team, our customer, or part of the wider world. We take promises seriously and deliver on them both internally and externally. We value our distributed work environment, and we trust our colleagues to make the best use of that flexibility to cultivate the best life balance for them.
+
+Some examples of this value in action are:
+
+- PEX and company credit cards; we trust people to purchase the things they need and give them freedom to do so.
+- We have a very liberal refund policy so that customers can decide what works for them.
+
 
 **Empower people**
 
-We learn by sharing our knowledge, and we help others do the same. We strive to create a space that empowers everyone to be and do what they want.
+We strive to create a space that empowers everyone. We encourage growth and exploration, with a culture of sharing and support for change. We give people the power to make choices about how to meet their needs and to change things about our company. We share as we learn, both with technology and business, to encourage others and assist with the lessons we’ve learned. 
+
+Some examples of this value in action are:
+
+- In addition to technical training information we encourage people to understand and use the wider array of tools that the open source community provides. 
+- We empower employees to be better for their family (parent, spouse, child) by giving them the flexibility to be there when and how they need to be.
```

Comment:
We give employees the flexibility to be there for their families when and how they need to be.

Reply:
How about something like: We allow employees to determine their own schedules to be there for their families when and how they need to be.

I want to convey the empowerment aspect of this.